app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class NoInternetEvent {
```

Comment:
So what do you suggest? We should handle all these error messages or just give a common one for errors and another one for internet not connected?